### DISCUSSION TITLE
Historical Record vs Opponent

### DISCUSSION THREAD

**Discussion by JDIacobbo** [nmelhado/league-page#159] [Ideas]
Would it be possible to implement a table to show historical records vs each opponent?

**Comment by nmelhado** [comment]
It should be. I think we could collect that info when we gather league records win/loss data and then we could display it on the individual manager pages.

I'm a little swamped right now, but if you make a feature request and link this discussion, maybe someone else could grab it.

This would definitely be a fun addition.
